source/osmaxx/settings.py

```diff
@@ -112,6 +190,31 @@
 #     os.path.join(BASE_DIR, 'resources/public/'),
 # )
 
- # login url if param 'next' is not set
+# login url if param 'next' is not set
 LOGIN_REDIRECT_URL = '/'
-LOGIN_URL = '/login'
+LOGIN_URL = '/excerptexport/login/'
+LOGOUT_URL = '/excerptexport/logout/'
+# TODO: show nice user error page
+SOCIAL_AUTH_LOGIN_ERROR_URL = '/excerptexport/'
+# Used to redirect the user once the auth process ended successfully. The value of ?next=/foo is used if it was present
+SOCIAL_AUTH_LOGIN_REDIRECT_URL = '/excerptexport/'
+# Is used as a fallback for LOGIN_ERROR_URL
+SOCIAL_AUTH_LOGIN_URL = '/excerptexport/'
+# Used to redirect new registered users, will be used in place of SOCIAL_AUTH_LOGIN_REDIRECT_URL if defined.
+SOCIAL_AUTH_NEW_USER_REDIRECT_URL = '/excerptexport/'
+# Like SOCIAL_AUTH_NEW_USER_REDIRECT_URL but for new associated accounts (user is already logged in).
+# Used in place of SOCIAL_AUTH_LOGIN_REDIRECT_URL
+SOCIAL_AUTH_NEW_ASSOCIATION_REDIRECT_URL = '/excerptexport/'
+# The user will be redirected to this URL when a social account is disconnected
+SOCIAL_AUTH_DISCONNECT_REDIRECT_URL = '/excerptexport/'
+# Inactive users can be redirected to this URL when trying to authenticate.
+# Successful URLs will default to SOCIAL_AUTH_LOGIN_URL while error URLs will fallback to SOCIAL_AUTH_LOGIN_ERROR_URL.
+SOCIAL_AUTH_INACTIVE_USER_URL = '/excerptexport/'
+SOCIAL_AUTH_LOGIN_SUCCESS_URL = '/excerptexport/'
```

Comment:
Is it possible to move the path to a variable instead of repeating it again and again? We will refactor the url structure soon and a variable will simplify that.